src/main/steps/applicant1/changing-to-a-sole-application/form.njk

```diff
@@ -0,0 +1,23 @@
+{% from "govuk/components/button/macro.njk" import govukButton %}
```

Comment:
this seems to be the same as the switch to sole form, maybe we can just include it.